### DISCUSSION TITLE
Running v3 model?

### DISCUSSION THREAD

**Discussion by pautri** [snakers4/silero-vad#310] [Q&A] [answered]
In discussion 268 it was mentioned that "v4 was trained not to respond to background voice". For a particular use case though I want all speech to be tagged, background or not. Is it possible to run the v3 model? If so, how?

**Comment by snakers4** [comment, marked as answer]
I believe the better way to do so it to tweak the hyper parameters of the VAD. It would still react to background speech to some extent.

**Reply by pautri** [reply]
thanks, I'll try that. Any suggestions in which direction? Lower min_speech_duration_ms and lower threshold?

**Reply by snakers4** [reply]
Just plot the probability chart using the provided utils and it will help you decide

**Reply by dgoryeo** [reply]
Hi @pautri , did you manage to find the right hyper-parameter settings for capturing background speeches?
